builds.yml

```diff
@@ -260,6 +260,10 @@ env:
   - TEST_GAS_FEE_FLOWS: false
   # Determines if feature flagged network ui new design
   - ENABLE_NETWORK_UI_REDESIGN: ''
+  # Determines if uses the security alerts API to validate confirmations
```

Comment:
Should we match mobile and use:

`Temporary mechanism to enable security alerts API prior to release`

Reply:
Done,  https://github.com/MetaMask/metamask-extension/pull/25544/commits/83a4bfeb4915f830494282e78f1f1ad5c715b793